op-e2e/system_test.go

```diff
@@ -1510,14 +1475,25 @@ func testFees(t *testing.T, cfg SystemConfig) {
 	ethPrivKey := cfg.Secrets.Alice
 	fromAddr := crypto.PubkeyToAddress(ethPrivKey.PublicKey)
 
-	// require.NotEqual(t, cfg.DeployConfig.L2OutputOracleProposer, fromAddr)
+	require.NotEqual(t, cfg.DeployConfig.ValidatorManagerTrustedValidator, fromAddr)
 	require.NotEqual(t, cfg.DeployConfig.BatchSenderAddress, fromAddr)
 
+	// Wait until Kroma MPT migration completed when it's set
+	if cfg.DeployConfig.L2GenesisKromaMPTTimeOffset != nil {
+		mptTimeOffset := *cfg.DeployConfig.L2GenesisKromaMPTTimeOffset
+		mptMigrationNextBlock := uint64(mptTimeOffset)/cfg.DeployConfig.L2BlockTime + 1
```

Comment:
Since the transition is applied from mptMigrationBlock, wouldn't it be more accurate to test by MPT block instead of next?